src/schematools/factories.py

```diff
@@ -10,7 +10,7 @@
 
 from geoalchemy2.types import Geometry
 from psycopg2 import sql
-from sqlalchemy import BigInteger, Boolean, Date, DateTime, Float, Numeric, String, Time
+from sqlalchemy import JSON, BigInteger, Boolean, Date, DateTime, Float, Numeric, String, Time
```

Comment:
I'm surprised isort is okay with this, but whatever.

Reply:
Yeah, I changed it manually.